tests/functional/simple_test.py

```diff
@@ -305,17 +312,19 @@ def flake8_newer():
 -r %s/requirements.d/coverage.txt
 ''' % TOP)
     venv_update()
-    assert pip_freeze() == '\n'.join((
-        'coverage==4.2',
+    frozen_requirements = pip_freeze().split('\n')
+    expected = (
+        'coverage==4.3.1',
         'coverage-enable-subprocess==1.0',
         'flake8==2.2.5',
         'mccabe==0.3',
         'pep8==1.5.7',
         'pyflakes==0.8.1',
         'venv-update==' + __version__,
-        'wheel==0.29.0',
-        ''
-    ))
+        '',
+    )
+    expected = maybe_add_wheel(expected)
+    assert set(frozen_requirements) == set(expected)
```

Comment:
I think all these edits can be reverted if you accept my proposal to remove maybe-add-wheel, below.